index.html

```diff
@@ -21,10 +24,11 @@
     <main class="landing">
         <div class="landiv">
             <h1>Hey there. I'm Abel<br><span class="greeny">I'm a software developer</span></h1>
-            <p id="main-support-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            <p id="main-support-text"> I can help you build a product , feature or website Look through some of my work
```

Comment:
- [OPTIONAL] Kindly use the font-family, font-size and color same as the Figma template.